src/appComponents/tracks/TracksController.js

```diff
@@ -2,55 +2,57 @@
  * Created by championswimmer on 29/5/15.
  */
 
-var tracksModule = angular.module('oe.tracks', ['ui.router']);
-
-tracksModule.config(['$stateProvider', function($stateProvider) {
-    $stateProvider.state('tracks', {
-        url: '/tracks',
-        templateUrl: 'appComponents/tracks/tracks.html',
-        controller: 'TracksController'
-    });
+const tracksModule = angular.module("oe.tracks", ["ui.router"]);
+
+tracksModule.config(["$stateProvider", function($stateProvider) {
+
+  $stateProvider.state("tracks", {
+    "url": "/tracks",
+    "templateUrl": "appComponents/tracks/tracks.html",
+    "controller": "TracksController"
+  });
+
 }]);
 
-tracksModule.controller('TracksController', 
-	['$mdDialog','$sessionStorage', '$rootScope', 'ApiJsonFactory',
-        function($mdDialog, $sessionStorage, $rootScope, ApiJsonFactory) {
-		var tc = this;
-        if ( $sessionStorage.tracks === null ||
-            typeof($sessionStorage.tracks) == 'undefined')
+tracksModule.controller("TracksController", ["$mdDialog","$sessionStorage", "$rootScope", "ApiJsonFactory",
+        function ($mdDialog, $sessionStorage, $rootScope, ApiJsonFactory) {
+          var tsc = this;
+
+          if ($sessionStorage.tracks === null || 
+            typeof $sessionStorage.tracks === "undefined")
         {
             $sessionStorage.tracks = [];
         }
-        tc.Tracks = $sessionStorage.tracks;
-        tc.Sessions = $sessionStorage.sessions;
+        tsc.Tracks = $sessionStorage.tracks;
+        tsc.Sessions = $sessionStorage.sessions;
         
 
-        if (tc.Tracks.length === 0) {
-            ApiJsonFactory.getJson('tracks')
+        if (tsc.Tracks.length === 0) {
+            ApiJsonFactory.getJson("tracks")
                 .then(function (response) {
-                    tc.Tracks = response.data.tracks;
-                    $sessionStorage.tracks = tc.Tracks;
+                    tsc.Tracks = response.data.tracks;
+                    $sessionStorage.tracks = tsc.Tracks;
                 }, function (error) {
-                    console.error(error);
+                    //console.error(error);
                 });
 
-            ApiJsonFactory.getJson('sessions')
+            ApiJsonFactory.getJson("sessions")
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 10 space characters but found 12.](https://www.codacy.com/app/triplez/open-event-webapp/file/2418113389/issues/source?bid=3233857&fileBranchId=3237740#l39)